attribute-event-requests/INDI-attribute.md

```diff
@@ -252,96 +278,254 @@ Found in the following historical records:
 ## Occupation
+### Description
+
 
+### Value
 In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Occupation`
 
+### Absence
+
+
+### Used
+- Part of the GEDCOM v7.0 Specification
+- Part of the GEDCOM X specification with URI http://gedcomx.org/Occupation
+- Used by the FamilySearch API via GEDCOM X
+
+--------------------------
 ## Physical Description
+### Description
+
 
+### Value
 In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/PhysicalDescription`
 
+### Absence
+
+
+### Used
+- Part of the GEDCOM v7.0 Specification
+- Part of the GEDCOM X specification with URI http://gedcomx.org/PhysicalDescription
+- Used by the FamilySearch API via GEDCOM X
+
+DESC.TYPE: can provide a user-language-specific indication of multiple aspects of an individual's physical description.  Enumerated example: {height, weight, eye color, tattoos, skin color, hair color, scars, lost limbs, other/phrase}.
+
+For example:
+```
+1 DESC Brown
+2 TYPE Hair Color
+```
+
+--------------------------
+## Property
+### Description
+
+
+### Value
+
+
+### Absence
+
+
+### Used
+- Part of the GEDCOM v7.0 Specification
+- Part of the GEDCOM X specification with URI http://gedcomx.org/Property
+- Used by the FamilySearch API via GEDCOM X
+
+--------------------------
 ## Religion
+### Description
+
 
+### Value
 In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Religion`
 
+### Absence
+
+
+### Used
+- Part of the GEDCOM v7.0 Specification
+- Part of the GEDCOM X specification with URI http://gedcomx.org/Religion
+- Used by the FamilySearch API via GEDCOM X
+
+--------------------------
 ## Residence
+### Description
 
+
+### Value
 In [GEDCOM X](https://github.com/FamilySearch/gedcomx/blob/master/specifications/fact-types-specification.md) with URI `http://gedcomx.org/Residence`
 
-## Single
+### Absence
 
-The assertion that a person is not a partner in a family can be made using the `NMR` structure with payload "`0`".
-This is distinct from simply not having any `FAMS` structures,
-which might mean there are relationships that have not yet been added to the data.
 
 ### Used
+- Part of the GEDCOM v7.0 Specification
+- Part of the GEDCOM X specification with URI http://gedcomx.org/Residence
+- Used by the FamilySearch API via GEDCOM X
 
-- Used by the [FamilySearch API](https://www.familysearch.org/developers/docs/guides/facts) with URI `http://familysearch.org/v1/NoCoupleRelationships`
+--------------------------
+## Single
+### Description
+The assertion that a person is not a partner in a family can be made using the `NMR` structure with payload "0". 
+This is distinct from simply not having any FAMS structures, which might mean there are relationships that have not yet been added to the data.
```

Comment:
I would think these would go under "Absence", since they're not a definition of what "Single" means/